features/models.py

```diff
@@ -257,6 +260,26 @@ def __str__(self):
         return self.id
 
 
+class PriceTag(TranslatableModel):
+    feature = models.ForeignKey(
+        Feature, on_delete=models.CASCADE, related_name="feature_price_tags"
```

Comment:
would it be neater give it `related_name=price_tags`, since it will be already on the `Feature` model? 🤔

Reply:
fixed